pages/common/octez-client.md

```diff
@@ -0,0 +1,32 @@
+# octez-client
+
+> Interact with the Tezos blockchain.
+> More information: <https://tezos.gitlab.io/user/setup-client.html>.
+
+- Configure the client with a connection to a Tezos RPC node such as `https://rpc.ghostnet.teztnets.com`:
```

Comment:
https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#heading
> All links must be enclosed inside angular brackets (`<` and `>`).

```suggestion
- Configure the client with a connection to a Tezos RPC node such as <https://rpc.ghostnet.teztnets.com>:
```

Reply:
This isn't really a link that you could click on; it's the URL to a server that you connect the client to, like you were going to send a `curl` request to a REST API. It's an RPC node, something like an HTTP API endpoint.

Reply:
Hi, that wouldn't be an issue since links and backticks are stylistically shown differently by our clients so even if it isn't clickable I would suggest using angular brackets here.

Reply:
@timothymcmackin can you please make the suggested changes as most of the maintainers agree so we can merge your PR?

Reply:
@spageektti Ok, I accepted the suggestion to use backticks here.